chapter_16_advanced_forms.asciidoc

```diff
@@ -385,6 +400,8 @@ We can do this because the database on our laptop is only used for dev, so the d
 In <<chapter_18_second_deploy>>, we'll deploy our new code to production,
 and discuss what to do if we run into migrations and data integrity issues at that point.
 
+// DAVID: Shall we get them to run migrations again, since they deleted the database?
```

Comment:
Update - we definitely should do this because otherwise when they try to start up the application in a subsequent chapter, it won't work.